Amplify/Categories/Auth/Models/AuthProvider.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 
+/// Supported auth providers
```

Comment:
Some context would make this more useful. What is an "auth provider"? When would a customer need to use it?